docs/Course Wiki/ECE Course Offerings/ECE330.md

```diff
@@ -0,0 +1,60 @@
+
+A basic understanding of electromagnetics concepts and Maxwell's equations is useful to understanding the theory behind magnetic circuits, transformers, and rotating machines.  Because of this, taking [ECE329](ECE329.md) - Fields and Waves I before or concurrently with this course can be helpful, but it is not essential to do well in the class.  Linear algebra concepts, such as determining eigenvalues, from either [MATH257](../MATH%20Course%20Offerings/MATH257.md) or [MATH416](../MATH%20Course%20Offerings/MATH416.md) are important for understanding linearization and electromechanical system stability, and it is recommended to take one of these courses either before or concurrently with ECE330.
+
+## When to Take it
+
+Take this course to satisfy the 3 out of 5 criteria, or if you have an interest in power systems, electric machines, or electromechanical energy conversion.  For students interested in the power and energy field, it is recommended to take this course as soon as possible, directly after taking [ECE210](ECE210.md).  However, it is completely fine to take this course later, as relevant concepts from ECE210 are reviewed during the first week.  If you want to take this course as soon as possible, take ECE210 as soon as possible.
+
+## Course Structure
+
+This course has a relatively light workload.  Weekly homeworks are assigned on PrairieLearn, which can take about a few hours.  The homework is not required, but is highly recommended in order to do well on the quizzes.  There are weekly quizzes that are similar to the homework problems.  These quizzes are short and should be manageable if you have completed the homework and understand the concepts behind the problems.  
+
+This course has two midterm exams and one final exam.  For the midterms, students can go through past exams and solutions. Past exams since Fall 1999 are uploaded on the course website. These are a great help when preparing for both quizzes as well as midterms.  For the final, it is highly recommended to go through every existing past exam deliberately.
+
+This course is graded on an absolute scale, meaning students are graded based on a performance standard, not their performance relative to other students, so doing well on the quizzes and exams is vital to doing well in the class.
+
+## Instructors
+
+Professors Jonathon Schuh, Olga Mironenko, and Richard Zhang have taught the course recently.  The professors all have strong backgrounds in the power field and are very easy to ask questions to.  Going to their office hours is recommended.
+
+## Course Tips
+
+Though it is not required, doing the homework is very important for doing well on the quizzes and exams.  Going to office hours and reading the textbook are helpful for completing the homework and studying for exams.  Many past exams are posted on the course website, and it is recommended for students to go through these exams and their homework problems to study for exams.
+
+## Life After
```

Comment:
For this section, format courses as follows:

[ECExxx - course title here](ECExxx.md)